lib/BC/InstructionLifter.cpp

```diff
@@ -290,9 +291,25 @@ InstructionLifter::LoadRegAddress(llvm::BasicBlock *block,
     reg_ptr_it->second = reg_ptr;
     return reg_ptr;
 
+  // Try to find it as a global variable.
+  } else if (auto gvar = module->getGlobalVariable(reg_name)) {
+    return gvar;
+
+  // Invent a fake one and keep going.
   } else {
```

Comment:
Drop `else` branch there is no reason to have it as previous branches always end with `return`.

Reply:
(See above)